src/tokens/tokens.service.ts

```diff
@@ -519,8 +538,8 @@ class TokenListener implements EventListener {
         : blockchainId + '/' + eventIndex.toString(10).padStart(6, '0');
```

Comment:
So I made this change but then realized that the "transfer ID" here really is somewhat similar to what we call a "protocol ID" in FireFly. It's a fully unique identifier for this transfer, derived from the block number and other info, specifically used for deduplication.

This is actually fundamentally different from the field on approvals now called "subject", which is not a fully unique identifier, but rather used to determine if an approval supercedes a previous one (so both are separate events, but the new approval takes effect and replaces the old one).

So perhaps I'm talking myself into thinking that the field on transfers should _not_ be "subject"..?